Grid.js

```diff
@@ -0,0 +1,168 @@
+import Cell from "./Cell.js";
+
+class Grid {
+    constructor(
+        gridSize,
+        numMines,
+    ) 
+    {
+        this.gridSize = gridSize;
+        this.numMines = numMines;
+        this.cells = [];
+    }
+
+    createGridCells() {
+        this.cells = new Array(this.gridSize);
+    
+        for (let i = 0; i < this.gridSize; i++) {
+            this.cells[i] = new Array(this.gridSize);
+        };
+    
+        for (let row = 0; row < this.gridSize; row++) {
+            for (let col = 0; col < this.gridSize; col++) {
+                let cell = new Cell(
+                    row,
+                    col,
+                    false,
+                    undefined,
+                    false,
+                    false
+                );
+    
+                this.cells[row][col] = cell;
+            };
+            
+        };
+    };
+
+    getCell(row, col) {
+        return this.cells[row][col];
+    }
+
+    changeCellProperty(row, col, prop, value){
+        this.cells[row][col][prop] = value;
+    }
+    
+    assignMines(){  
+        function getRandomInt(max) {
+            return Math.floor(Math.random() * max);
+        }
+        for (let i = 0; i < this.numMines;) {
+            let row = getRandomInt(this.gridSize);
+            let col = getRandomInt(this.gridSize);
+            let cell = this.getCell(row, col);
+            if (!cell.isMine) {
+                cell.isMine = true;
+                i++;
+            }
+        }
+    };
+
+    applyFnToNeighbors(row, col, callback){
+        let topLeft = {row: row-1, col: col-1};
+        let topCenter = {row: row-1, col: col};
+        let topRight = {row: row-1, col: col+1};
+        
+        let centerLeft = {row: row, col: col-1};
+        let centerRight = {row: row, col: col+1};
+    
+        let bottomLeft = {row: row+1, col: col-1};
+        let bottomCenter = {row: row+1, col: col};
+        let bottomRight = {row: row+1, col: col+1};
+    
+        let surroundingCells = [topLeft, topCenter, topRight, 
+                                centerLeft, centerRight, 
+                                bottomLeft, bottomCenter, bottomRight];
+        //console.log(surroundingCells);
+        
+        surroundingCells.forEach(c => callback(c));
```

Comment:
I would name this `fn` since that's in your function name, makes a bit more sense.
```suggestion
    applyFnToNeighbors(row, col, fn) {
        let topLeft = {row: row-1, col: col-1};
        let topCenter = {row: row-1, col: col};
        let topRight = {row: row-1, col: col+1};
        
        let centerLeft = {row: row, col: col-1};
        let centerRight = {row: row, col: col+1};
    
        let bottomLeft = {row: row+1, col: col-1};
        let bottomCenter = {row: row+1, col: col};
        let bottomRight = {row: row+1, col: col+1};
    
        let surroundingCells = [topLeft, topCenter, topRight, 
                                centerLeft, centerRight, 
                                bottomLeft, bottomCenter, bottomRight];
        
        surroundingCells.forEach(c => fn(c));
```

Reply:
consistent style - talked about